HHS/Head-Start-TTADP
# [TTAHUB-1759] Training report form

## Description of change
This PR includes
- all instances of react-hook-form have been bumped to use the non IE-11 distro
- a new permission was added "COLLABORATOR_TRAINING_REPORTS"
- Navigator component refactored to activity-report-agnostic
- First page of form created ("event summary")

This does not include:
- exhaustive permissions. we can rely on the feature flag for now, as this PR is already large enough
- good test coverage. will wait till more functionality is complete to add additional tests, like e2e 

I also understand that this may need to be revised once #1558 is merged. Planning on crossing that bridge when I come to it.

## How to test
- This PR includes updates to the ```@ttahub/common``` package, so you'll need to reinstall that
- Import training events via the importCLI job (you may have to modify the script with an existing email if you aren't using the test data)
- You may also need to add users with the new permissions
- View the form at /training-report/{:trainingReportId}
- Note the imported fields, make changes, save draft. Confirm that the handlers work.
- Smoke test the Activity report form
- **The main thing to test on this PR is that the ActivityReport form remains intact and functional without any new bugs.**

Note also - as I've stated, due to the size of this PR, you should expect things to be incomplete. Review #1577 for a more functional version of this. This is intended to be a first step, the separation of the Navigator component from the AR without breaking anything.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1759
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1769


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
